qutebrowser/config/configdata.yml

```diff
@@ -503,13 +503,13 @@ content.host_blocking.whitelist:
   - piwik.org
   type:
     name: List
-    valtype: String
+    valtype: QUrl
     none_ok: true
   desc: >-
-    List of domains that should always be loaded, despite being ad-blocked.
+    List of URL Patterns should always be loaded, despite being ad-blocked.
 
-    Domains may contain * and ? wildcards and are otherwise required to exactly
-    match the requested domain.
+    Patterns may contain the * wildcard and are based on chromium url matching
+    guidelines. See utils.urlmatch.py for examples.
```

Comment:
Do we actually still need that part? I think if we describe how patterns work, we should use the same description for all URL pattern config values for consistency. Currently, I only see `completion.web_history.exclude` which doesn't have any description of the patterns. But that also uses `UrlPattern` while this (and `content.host_blocking.whitelist`) (currently) use `QUrl` as `List` `valtype`. I really think the difference is not clear from the current descriptions.

Reply:
Note that during my first review, the `valtype` to was changed to `QUrlPattern`. Also I might have confused the settings I was referring to. So here the correction:

* The new setting `content.host_blocking.whitelist` now uses `QUrlPattern` and the description

> Patterns may contain the * wildcard and are based on chromium url matching match the requested domain.		     guidelines. See utils.urlmatch.py for examples.

* The existing option `completion.web_history.exclude` uses `UrlPattern` (no `Q`!) and does not explain the pattern syntax.

Whatever we decided, I think these to settings should either

1. Use the same `valtype` and matching descriptions, or
2. have a paragraph in their description that explains the difference(s).

Reply:
I have fixed the valtype so they both use UrlPattern, and updated the description of content.host_blocking.whitelist to remove the excess information about pattern matching to fit with the other config values.